### PR TITLE
Add initial example

### PR DESCRIPTION
**Pull request by dashersw**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// implement a fizz buzz function.
+
+function fizzBuzz(n) {
+  var result = []
```
Comment (dashersw) [inline index.js:4]: this should be const

Comment (dashersw) [inline index.js:4]: this should be a const
```suggestion
  const result = []
```

Comment (dashersw) [reply index.js:4]: Here are some code suggestions:
```suggestion
  helloWorld();
```

Comment (dashersw) [inline index.js:4]: prefer const over var

Comment (dashersw) [reply index.js:4]:
```suggestion
-  var result = []
+  const result = []
```

Comment (dashersw) [inline index.js:4]:
```suggestion
// implement a fizz buzz function.

function fizzBuzz(n) {
  const result = []
```

File: index.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  for (var i = 1; i <= n; i++) {
```
Comment (dashersw) [inline index.js:5]: use const and let instead of var.

Comment (felix-weizman-deel) [inline index.js:5]: use const and let instead of var.

[the remark this answers is not in the document]
Comment (dashersw) [reply index.js:5]: Here are some code suggestions:
```suggestion
  helloWorld();
```

File: index.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    if (i % 15 === 0) result.push('FizzBuzz')
```
Comment (felix-weizman-deel) [inline index.js:6]: test

File: index.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    else if (i % 3 === 0) result.push('Fizz')
+    else if (i % 5 === 0) result.push('Buzz')
+    else result.push(i.toString())
```
Comment (felix-weizman-deel) [inline index.js:9]: Test

File: index.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+  }
```
Comment (felix-weizman-deel) [inline index.js:10]: test

**Pull request closed without merging**